__tests__/CountValidator.test.js

```diff
@@ -0,0 +1,35 @@
+import CountValidator from '../src/utils/validator/CountValidator.js';
+import ErrorMessage from '../src/constants/ErrorMessage.js';
+
+test('시도횟수가 숫자가 아니면 에러 처리한다.', () => {
+  const TRY_COUNT = NaN;
+
+  expect(() => CountValidator.isNumber(TRY_COUNT)).toThrow(
+    ErrorMessage.isNotNumber,
+  );
+});
+
+// todo: test.each
+test('시도횟수가 0 이하이면 에러 처리한다.', () => {
+  const TRY_COUNT = 0;
+
+  expect(() => CountValidator.isAtLeastOne(TRY_COUNT)).toThrow(
+    ErrorMessage.isInvalidNumber,
+  );
+});
+
+test('시도횟수가 정수가 아니면 에러 처리한다.', () => {
+  const TRY_COUNT = 0.5;
+
+  expect(() => CountValidator.isInteger(TRY_COUNT)).toThrow(
+    ErrorMessage.isNotInteger,
+  );
+});
+
+test('시도횟수가 100만번을 넘으면 에러 처리한다.', () => {
+  const TRY_COUNT = 2000000;
```

Comment:
요런 테스트를 할 때는 무작정 더 큰 숫자보다는 경계값에 대한 테스트를 해보시면 좋을거 같아요!
```suggestion
test('시도횟수가 100만번을 넘으면 에러 처리한다.', () => {
  const TRY_COUNT = 1000001;
```

Reply:
'경계값'이라는 개념에 대해 알지 못했는데, 조언 감사합니다! 피드백을 반영하여 수정해보겠습니다! ㅎㅎ